cmd/livepeer/livepeer.go

```diff
@@ -434,7 +488,7 @@ func main() {
 			// of RoundsManager that reads from a cache
 			// TODO: Initialize Sender with an implementation
 			// of SenderManager that reads from a cache
-			n.Sender = pm.NewSender(n.Eth, n.Eth, n.Eth, ev, *depositMultiplier)
+			n.Sender = pm.NewSender(n.Eth, n.Eth, senderWatcher, ev, *depositMultiplier)
```

Comment:
Should the TODO note before this line be updated?

🎉 for starting to differentiate the NewSender parameters :)

Reply:
Yes it should, missed that !